microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md

```diff
@@ -0,0 +1,180 @@
+
+:::image type="content" source="../../media/2021/11/console-displayed-when-elements-tab-selected.png" alt-text="When a tool other than the Console is selected in the main toolbar, the Drawer opens and displays the Console.":::
+
+When **Console** is selected in the top panel, the bottom panel is automatically minimized, if the bottom panel was displaying the **Console**:
+
+:::image type="content" source="../../media/2021/11/console-hidden-when-console-tab-selected.png" alt-text="When Console is selected in the main toolbar, the Drawer is automatically minimized, if the Drawer was displaying the Console.":::
+
+If you want to only allow the Console tool to open in a single place, you can configure the behavior of the Console.  In the main toolbar or the **Drawer** toolbar, right-click the **Console** tab, and then select **Configure console**.  The **Settings** > **Preferences** page is displayed.  In the **Console** section, clear the checkbox **Show Console tab in main panel and drawer**, then click **Close** (**x**).
+
+<!-- video: see work item -->
+
+
+<!-- ====================================================================== -->
+## Microsoft Edge DevTools Visual Studio Code extension
+
+<!-- paragraph to prevent h2 immediately followed by h3: -->
+For general information about this extension, see [Microsoft Edge DevTools extension for Visual Studio Code](../../../../visual-studio-code/microsoft-edge-devtools-extension.md).
+
+<!-- 3 -->
+
+<!-- Title: Dockable browser screencast, device emulation, and live issue reporting, and in Microsoft Edge DevTools for Visual Studio Code -->
+<!-- Subtitle: Display your web project inside the editor, simulate different devices, and get notified about issues with your code while you develop it. --> 
+
+### Display your web project inside the editor, and simulate different devices
+
+The Microsoft Edge DevTools for Visual Studio Code extension now features a dockable screencast and device emulation:
+
+:::image type="content" source="../../media/2021/11/edge-devtools-for-vscode-toggle-screencast.png" alt-text="Click the screencast icon to view the browser inside of Visual Studio Code.":::
+
+You can see your web project in a dedicated tab inside of Visual Studio Code, and also simulate various devices:
+
+:::image type="content" source="../../media/2021/11/edge-devtools-for-vscode-simulated-iphone-red-boxes.png" alt-text="The screencast showing the current document in an emulated iPhone 5 in the correct size and with an emulated touchscreen interface.":::
+
+<!-- videos: see work item -->
+
+For details about the dockable screencast, see [Changing to headless mode](../../../../visual-studio-code/microsoft-edge-devtools-extension.md#changing-to-headless-mode).  For details about device emulation, see [Device emulation in the screencast](../../../../visual-studio-code/microsoft-edge-devtools-extension.md#device-emulation-in-the-screencast).
```

Comment:
Headless mode is a different thing than the screencast, I opt for removing the first sentence.